### PR TITLE
changes in jump for snowball flagging and flagging pixels after detected jump

### PR DESCRIPTION
**Pull request by cshanahan1**
Resolves: RCAL-410

Required changes for new keywords in jump step that control snowball and shower flagging. Since this is not being used for roman currently, i have not added any tests here (it is tested in stcal and jwst).

### CODE REVIEW

**Comment by nden** [comment]
@cshanahan1 There are code style issues preventing tests from running

File: romancal/jump/jump_step.py [hunk 1]
```diff
@@ -79,6 +85,12 @@ def process(self, input):
             max_jump_to_flag_neighbors = self.max_jump_to_flag_neighbors
             min_jump_to_flag_neighbors = self.min_jump_to_flag_neighbors
             flag_4_neighbors = self.flag_4_neighbors
+            min_sat_area = self.min_sat_area
+            min_jump_area = self.min_jump_area
+            expand_factor = self.expand_factor
+            use_ellipses = self.use_ellipses
+            sat_required_snowball = self.sat_required_snowball
+            expand_large_events = self.expand_large_events
```
Comment (nden) [inline romancal/jump/jump_step.py:93]: I'm not sure it's necessary to assign to separate variables here but I se it foloows what the rest of the step is doing.

**Review by nden — APPROVED** [review]
LGT

**Review by ddavis-stsci — APPROVED** [review]
LGTM

**Pull request merged**